Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/ProjectSettings/UnityAdsSettings.asset

```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!292 &1
+UnityAdsSettings:
```

Comment:
None of those examples has a separate project settings so let's be consistent till we can :) I would recommend removing the project settings folder ... feel free to use different folders in there to organize the files. This will also keep things cleaner.